### PR TITLE
Add notes about flux bootstrap and feature parity

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
LGTM 💯

File: docs/guides/flux-v1-migration.md [hunk 1]
```diff
@@ -53,6 +62,13 @@ to define the state of your fleet of Kubernetes clusters.
 
 For a detailed walk-through of the bootstrap procedure please see the [installation guide](installation.md).
 
+!!! warning "`flux bootstrap` target"
```
Comment (squaremo) [inline docs/guides/flux-v1-migration.md:65]: :+1:

File: docs/guides/flux-v1-migration.md [hunk 2]
```diff
@@ -6,7 +6,16 @@ Read the [FAQ](../faq/index.md) to find out what differences are between v1 and
...
-    
+
+!!! info "Feature parity"
+    "Feature parity" does not mean Flux v2 works exactly the same as v1 (or is
+    backward-compatible); it means you can accomplish the same results, while
+    accounting for the fact that it's a system with a substantially different
+    design.
+    This may at times mean that you have to make adjustments to the way your
+    current cluster configuration is structured. If you feel this is you,
+    and you need help, please refer to the [support page](https://fluxcd.io/support/).
```
Comment (squaremo) [inline docs/guides/flux-v1-migration.md:17]:
```suggestion
    current cluster configuration is structured. If you are in this situation and
    need help, please refer to the [support page](https://fluxcd.io/support/).
```

"this" referring to the implicit situation of having to adjust configuration is a bit unclear; the above makes it explicit. Could also cut out some verbiage with

```suggestion
    current cluster configuration is structured. If you need help with this, please refer to the [support page](https://fluxcd.io/support/).
```

**Review by squaremo — APPROVED** [review]
Thanks Hidde. I would be happy with this as-is; I've left a modest suggestion.

File: docs/guides/flux-v1-migration.md (hunk 2 above)
Comment (hiddeco) [reply docs/guides/flux-v1-migration.md]: To me the verbiage sounded more friendly and/or sympathetic, so I opted for suggestion 1. Thank you :100:

**Pull request merged**